src/pybind/mgr/cephadm/services/iscsi.py

```diff
@@ -23,6 +23,14 @@ def config(self, spec: IscsiServiceSpec) -> None:  # type: ignore
         assert spec.pool
         self.mgr._check_pool_exists(spec.pool, spec.service_name())
 
+    def build_trusted_ip_list(self, spec: IscsiServiceSpec) -> str:
```

Comment:
FWIW I personally find it confusing when a word in the function "disagrees" with the return type. In this example the suffix "_list" implies to me a python list type, but in reality it's returning a string. Sure, the string is a "comma separated list" but therein lies my confusion.  It's not a dealbreaker but it might be nice to choose a different name. On the other hand maybe it's better to leave it as-is to be consistent with the spec's attribute :shrug:

Reply:
I've changed it to the slightly more ambiguous but at least not directly deceitful `get_trusted_ips`. Thoughts?